cmd/ctr/commands/images/pull.go

```diff
@@ -89,6 +91,10 @@ command. As part of this process, we do the following:
 			Name:  "sync-fs",
 			Usage: "Synchronize the underlying filesystem containing files when unpack images, false by default",
 		},
+		&cli.BoolFlag{
+			Name:  "force",
+			Usage: "Force pulling by removing existing image and ALL its content blobs (including layers shared with other images) before re-downloading from registry. Use with caution.",
```

Comment:
can you explain the with caution comment. user may not understand this means shared layers will  need to be downloaded before new containers with the shared content can be created..